[ Backported.  ]

commit 684341392f3ca6703dc28dac548d3051811bff47
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Tue Mar 30 18:37:03 2010 +0000

    Add org.gnu.gdb.i386.avx.
    
    2010-03-30  H.J. Lu  <hongjiu.lu@intel.com>
    
    	* gdb.texinfo (i386 Features): Add org.gnu.gdb.i386.avx.

--- gdb-7.1-p6/gdb/doc/gdb.texinfo	2010-04-03 21:09:29.000000000 +0200
+++ gdb-7.1/gdb/doc/gdb.texinfo	2010-04-03 21:10:57.000000000 +0200
@@ -33380,6 +33380,17 @@ describe registers:
 @samp{mxcsr}
 @end itemize
 
+The @samp{org.gnu.gdb.i386.avx} feature is optional.  It should
+describe the upper 128 bits of @sc{ymm} registers:
+
+@itemize @minus
+@item
+@samp{ymm0h} through @samp{ymm7h} for i386
+@item
+@samp{ymm0h} through @samp{ymm15h} for amd64
+@item 
+@end itemize
+
 The @samp{org.gnu.gdb.i386.linux} feature is optional.  It should
 describe a single register, @samp{orig_eax}.
 
